src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIOffenderService.kt

```diff
@@ -86,4 +106,20 @@ class CommunityAPIOffenderService(
       .block()
       ?.staffIdentifier
   }
+
+  fun getResponsibleOfficer(crn: String): ResponsibleOfficer {
+    val offenderManagersPath = UriComponentsBuilder.fromPath(offenderManagersLocation)
+      .buildAndExpand(crn)
+      .toString()
+
+    // technically this endpoint can return more than one offender manager that
+    // is a responsible officer. in practice this should not happen. if it does
+    // we just take the first one we come across.
+    return communityApiClient.get(offenderManagersPath)
+      .retrieve()
+      .bodyToFlux(OffenderManagerResponse::class.java)
+      .filter { it.isResponsibleOfficer == true }
+      .map { ResponsibleOfficer(stringUtils.joinNullable(it.staff?.forenames, it.staff?.surname), it.staff?.email) }
+      .blockFirst()
```

Comment:
Given the amount of data-related assumptions we had before, could we pop up a "our assumption about the data was invalid" telemetry here if there are `!= 1` matches?

Feels like this would make our assumptions explicit and monitorable

```kt
telemetryClient.trackEvent(
  "InterventionsDataAssumptionInvalid",
  mapOf(
    "message": "expected only one responsible officer",
    "got": "2",
    // plus CRN and referralId
  ),
  null
)
```

Reply:
done